dogs.html

```diff
@@ -0,0 +1,193 @@
+            </div>
+          </div>
+        </div>
+        <div class="grid-main9">
+          <div class="dog-box">
+            <img src="./images/angus-card.jpg" alt="fluffy white small dog running toward viewer">
+            <div>
+              <h2>Angus</h2>
+              <p><span class="p-bold">Cost to Adopt:</span> $123.45</p>
+              <p>Shoober waggy wags vvv shoober shoob, ruff dat tungg tho blep. Very taste wow floofs shooberino long water shoob, long woofer bork.</p>
+              <button type="submit">Adopt</button>
+            </div>
+          </div>
+        </div>
+        <div class="grid-main10">
+          <div class="dog-box">
+            <img src="./images/violet-card.jpg" alt="fluffy white small dog running toward viewer">
+            <div>
+              <h2>Violet</h2>
+              <p><span class="p-bold">Cost to Adopt:</span> $123.45</p>
+              <p>Shoober waggy wags vvv shoober shoob, ruff dat tungg tho blep. Very taste wow floofs shooberino long water shoob, long woofer bork.</p>
+              <button type="submit">Adopt</button>
+            </div>
+          </div>
+        </div>
+        <div class="grid-main12">
+          <div class="dog-box">
+            <img src="./images/piper-card.jpg" alt="fluffy white small dog running toward viewer">
+            <div>
+              <h2>Piper</h2>
+              <p><span class="p-bold">Cost to Adopt:</span> $123.45</p>
+              <p>Shoober waggy wags vvv shoober shoob, ruff dat tungg tho blep. Very taste wow floofs shooberino long water shoob, long woofer bork.</p>
+              <button type="submit">Adopt</button>
+            </div>
+          </div>
+        </div>
+        <div class="grid-main13">
+          <div class="dog-box">
+            <img src="./images/maximus-card.jpg" alt="fluffy white small dog running toward viewer">
+            <div>
+              <h2>Maximus</h2>
+              <p><span class="p-bold">Cost to Adopt:</span> $123.45</p>
+              <p>Shoober waggy wags vvv shoober shoob, ruff dat tungg tho blep. Very taste wow floofs shooberino long water shoob, long woofer bork.</p>
+              <button type="submit">Adopt</button>
+            </div>
+          </div>
+        </div>
+        <div class="grid-main14">
+          <div class="dog-box">
+            <img src="./images/luna-card.jpg" alt="fluffy white small dog running toward viewer">
+            <div>
+              <h2>Luna</h2>
+              <p><span class="p-bold">Cost to Adopt:</span> $123.45</p>
+              <p>Shoober waggy wags vvv shoober shoob, ruff dat tungg tho blep. Very taste wow floofs shooberino long water shoob, long woofer bork.</p>
+              <button type="submit">Adopt</button>
+            </div>
+          </div>
+        </div>
+        <div class="grid-main15">
+          <div class="dog-box">
+            <img src="./images/Stella-card.jpg" alt="fluffy white small dog running toward viewer">
```

Comment:
This image isn't displaying. Thinking you may have a small typo with the image name